data/analytics/attributes.yml

```diff
@@ -185,11 +225,35 @@
   type: string
   redact: false
 
+- name: navigation_list_type
+  description: Used in the page_view object to record the navigation list type
```

Comment:
Can we explain any more? This is in the browse, right? Could we something like `Used in the page view object to record the type of page being shown in browse, either 'curated' or 'alphabetical'`? (I think the alternative is alphabetical, but check).

Reply:
@andysellick Will update it but I'll have to take your word for it, I can't find anywhere where it gives an example of the values, I can only find the pages from the trello card which have `curated` as the value!